src/rawchannel.cc

```diff
@@ -599,68 +675,133 @@ class RawChannel : public Nan::ObjectWrap
   {
     Nan::HandleScope scope;
 
-    struct can_frame frame;
-
+    struct canfd_frame framefd;         // for CANFD frame
+    struct can_frame frame;             // for CAN frame
+    
     unsigned int framesProcessed = 0;
+    
+    if (Flag_CANFD_Used){               // standard CAN frame
```

Comment:
as i understood candump implementation canfd_frame can store also classic frames too, so no need for if(canfd) else ... approach